targets/nintendo-switch.json

```diff
@@ -0,0 +1,31 @@
+{
+  "llvm-target": "aarch64",
+  "build-tags": ["nintendoswitch", "arm64"],
+  "goos": "linux",
+  "goarch": "arm64",
+  "compiler": "clang",
+  "linker": "ld.lld",
+  "rtlib": "compiler-rt",
+  "libc": "picolibc",
+  "gc": "conservative",
+  "relocation-model": "pic",
+  "cflags": [
+    "-target", "aarch64-none-linux-gnu",
+    "-mtune=cortex-a53",
+    "-g",
```

Comment:
@racerxdl see https://github.com/tinygo-org/tinygo/pull/1225 do you need to remove the `-g` flag from here?

Reply:
Oh, I think it was a copy-paste issue. Fixing it.